_data/jackleslie/jackleslie.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Jack Leslie
+institution: University of St Andrews
+profile_pic: jackleslie.png
+quote: "Aw, Man!"
```

Comment:
Please follow the given [instructions](https://github.com/education/graduation#3-add-your-profile-information) by removing the quotes to maintain the same format